docs/source/_toctree.yml

```diff
@@ -1,68 +1,70 @@
 - local: index
   title: 🤗 Transformers.js
 - sections:
-  - local: installation
-    title: Installation
-  - local: pipelines
-    title: The pipeline API
-  - local: custom_usage
-    title: Custom usage
+    - local: installation
```

Comment:
Can you maybe fix the indentation here? Just to lower the diffs.

Reply:
yes I'll revert and modify only what's is needed. as you notice there is some weird mismatches on my auto formatting, not sure where did the tabspaces came from